discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -130,19 +130,12 @@ def select_installation_disk(self, hosts_with_disk_paths):
 
     def set_ocs(self, ocs_enabled):
         logging.info(f'Enabling Ocs to:{ocs_enabled} for cluster: {self.id}')
-        self.set_operator('ocs', ocs_enabled)
-
-    def set_cnv(self, cnv_enabled):
-        logging.info(f'Enabling CNV to:{cnv_enabled} for cluster: {self.id}')
-        self.set_operator('cnv', cnv_enabled)
-
-    def set_operator(self, operator, enabled):
         cluster = self.api_client.cluster_get(self.id)
         olm_operators = cluster.olm_operators
         olm_names = [olm_operator['name'] for olm_operator in olm_operators]
-        if operator not in olm_names and enabled:
+        if 'ocs' not in olm_names and ocs_enabled:
             olm_operators.append({'name': operator})
```

Comment:
we can update the cluster as: `self.api_client.update_cluster(self.id, {"olm_operators":[{"name":"ocs"}]}) `

Reply:
For the tests, I guess we only need 1 operator to be enabled at a time, hence to enable, we could do  
``` self.api_client.update_cluster(self.id, {"olm_operators":[{"name":"ocs"}]}) ```
and to disable we could do:
```self.api_client.update_cluster(self.id, {"olm_operators":[]}) ```

Reply:
Or if multiple operators are to be enabled, then your way of updating operators is also fine.

Reply:
we might be looking at a combination of CNV and OCS, so having these convenience functions is \_a_good_thing_ :smile: